code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -4092,4 +4092,32 @@
 	taste_mult = 2
 	reagent_state = LIQUID
 	nutriment_factor = 40 //very filling
-	color = "#d169b2"
\ No newline at end of file
+	color = "#d169b2"
+
+//ADDITIONS BELOW THIS LINE MADE ON 02/28/2021
+
+/datum/reagent/drink/soda/kompot
+	name = "Kompot"
+	id = "kompot"
+	description = "A traditional eastern european beverage once used to preserve fruit in the 1980s"
```

Comment:
`Eastern European` I think. Not completely sure if `eastern` is part of the noun or not.